gradle.properties

```diff
@@ -24,7 +24,7 @@ parchment=2023.09.03
 
 badPackets=0.4.3
 WTHIT=8.6.0
-rareCandy=2.4.29-SNAPSHOT
+rareCandy=2.5.0-SNAPSHOT
```

Comment:
if this is a major update we should bump rarecandy to 3.x.x

Reply:
3.0.0 is hydos's RKS branch. For I just switched to backporting the assimp code.

Reply:
ok